tests/conftest.py

```diff
@@ -0,0 +1,26 @@
+import pytest
```

Comment:
Can you explain a bit more what this is for ?

Reply:
More info regarding the tool you can find under: https://docs.pytest.org/en/latest/
conftest.py is used to configure environment. There is a definition like "fixture" which will automatically run according to "scope" that defined for it or by default scope.